packages/stark-ui/src/modules/session-ui/pages/preloading/preloading-page.component.html

```diff
@@ -0,0 +1,22 @@
+<div class="stark-session-ui-page" role="alertdialog" aria-busy="true" aria-live="assertive">
+	<!-- adds "translate-cloak" class while loading translations -->
+	<header><i></i></header>
+	<content>
+		<div *ngIf="!userFetchingFailed">
+			<div class="stark-loading-icon"></div>
+			<h1 translate>STARK.PRELOADING.FETCHING_USER_PROFILE</h1>
+		</div>
+		<div *ngIf="userFetchingFailed">
+			<h2 translate>STARK.PRELOADING.FETCHING_USER_PROFILE_FAILURE</h2>
+			<h2 translate>STARK.PRELOADING.CONTACT_IT_SUPPORT</h2>
+			<h3><span translate>STARK.PRELOADING.CORRELATION_ID</span>: {{ correlationId }}</h3>
+			<button mat-button
+					mat-raised-button
+					color="primary"
+					(click)="reload()"
+					[attr.aria-label]="'STARK.PRELOADING.RELOAD' | translate">
+				<span translate>STARK.PRELOADING.RELOAD</span>
+			</button>
+		</div>
+	</content>
```

Comment:
The spinner is not shown in the preloading page as before :(
![preloading without spinner](https://user-images.githubusercontent.com/5698318/46013410-81e61100-c0cc-11e8-938f-33f38f1ab003.PNG)